lte/cloud/go/services/policydb/obsidian/handlers/policy_handlers_test.go

```diff
@@ -1194,9 +1232,9 @@ func TestPolicyWithQoSProfile(t *testing.T) {
 	// Delete profile
 	tc = tests.Test{
 		Method:         "DELETE",
-		URL:            "/magma/v1/lte/n1/policy_qos_profiles/profile0",
+		URL:            "/magma/v1/lte/n1/policy_qos_profiles/profile1",
```

Comment:
shouldn't you also delete `profile0`?

Reply:
It's not necessary for the test logic since it's no longer associated to the policy after the PUT from line 1195. (I don't think the DELETEs are needed to clean up test data, the test generates other entities that are not removed at the end.)

Reply:
Thanks, that clarifies it